tutorial.md

```diff
@@ -238,6 +215,42 @@ sudo mkdir -p /opt/cni/bin
 sudo cp bin/* /opt/cni/bin/
 ```
 
+Plugins
+
+```
+cd $GOPATH/src/github.com/containernetworking/plugins
+```
+
+Build the `plugin` binaries:
+
+```
+./build.sh
+```
+
+Output:
+
+```
+Building plugins
+  flannel
+  portmap
+  tuning
+  bridge
+  ipvlan
+  loopback
+  macvlan
+  ptp
+  vlan
+  dhcp
+  host-local
+  sample
+```
+
+Install the `plugin` binaries:
+
+```
+sudo cp bin/* /opt/cni/bin/
+```
+
```

Comment:
And that section is not needed. It's already described in the `CNI plugins` section.